apps/server/src/services/RestoreService.ts

```diff
@@ -0,0 +1,106 @@
+import { Playback } from 'ontime-types';
+import { logger } from '../classes/Logger.js';
+import { PlaybackService } from './PlaybackService.js';
+import { eventTimer } from './TimerService.js';
+import { EventLoader } from '../classes/event-loader/EventLoader.js';
+
+//File stuff
+import fs from 'fs';
+import { resolveDbDirectory } from '../setup.js'
+import path from 'path';
+import { Writer } from 'steno';
+
+type OntimeState = {
+    playback: Playback;
+    selectedEventId: string | null;
+    startedAt: number | null;
+    addedTime: number | null;
+    pausedAt: number | null
+}
+
+/**
+ * Service manages saveing of timer state
+ * that can then be resored when reopening
+ */
+export class RestoreService {
```

Comment:
this class should be exported since we export it initialised as a singleton here

also: could you add a few tests to the constructor and restore functions?

Reply:
> this class should be exported since we export it initialised as a singleton here

Sorry I am not understanding the grammar here i think 🇩🇰 

I will look in to some testing

Reply:
Hahaha. No need to apologise, my spaghetti English.

I meant that we didn't need to export the class. Sorry for the poorly verbalised comment